app.py

```diff
@@ -33,13 +29,45 @@
 @app.route("/bibliography")
 @requires_db
 async def list_bibliographies(db, request):
-    to_return = [{
-        "eid": bib.eid,
-        "description": bib.description,
-        "created": bib.created,
-        "modified": bib.modified
-    } for bib in Bibliography.list(db)]
-    return json(to_return)
+    bibliographies = [
+        {
+            "eid": bib.eid,
+            "description": bib.description,
+            "created": bib.created,
+            "modified": bib.modified
+        }
+        for bib in Bibliography.list(db)
+    ]
+    return json(bibliographies)
+
+
+@app.route("/bibliography_eid")
+@requires_db
+async def bibliography(db, request):
+    """
+    Blibliography associated with a blibliography eid
+    """
+    bibliography_eid = request.args.get('eid', None)
+    if not bibliography_eid:
+        return json(
+            {
+                'error': 'You must suply a bibliography eid.',
+                'details': 'Fill in the bibliography field.'
+            },
+            status=400
+        )
+
+    bibliography = [
+        {
+            "eid": bib.eid,
+            "description": bib.description,
+            "created": bib.created,
+            "modified": bib.modified
+        }
+        for bib in Bibliography.list(db)
+        if bib.eid == bibliography_eid
```

Comment:
Let's use this function instead http://home.condor-ir.co/condor.models.html#condor.models.base.AuditableMixing.find_by_eid, it would read, `Bibliography.find_by_eid(db, eid)`, it could return `None` in that case, just throw down a `404`.